src/hitch.c

```diff
@@ -2768,25 +2797,36 @@ drop_privileges(void)
 
 int
```

Comment:
static?

Reply:
you'd like `void drop_privileges(void)` to be static?

Reply:
I'm referring to `get_backaddr_info()`.

That said, `drop_privileges` should probably be too. :-)